crates/aptos-rosetta/src/types/objects.rs

```diff
@@ -987,8 +986,6 @@ pub struct CoinEvent {
 
 #[derive(Clone, Debug, Deserialize)]
 pub struct SetOperatorEvent {
-    _pool_address: Address,
-    _old_operator: Address,
```

Comment:
This is what was breaking reading, because the unused arguments don't match the serialization string

Reply:
Why though? The SetOperatorEvent emitted on chain does have pool_address and old_operator

Reply:
This is just the JSON serialization derive in rust.  It's trying to deserialize "_pool_address".  I don't need those two fields, so I just dropped them for now.